modules/iam-group/main.tf

```diff
@@ -0,0 +1,82 @@
+locals {
+  action = var.type == "read-only" ? local.read_only_action : var.type == "admin-access" ? local.admin_access_action : var.type == "other" ? var.police_action : []
+
+  admin_access_action = ["*"]
+  read_only_action = [
+    "aws-portal:ViewBilling",
+    "aws-portal:ViewUsage",
+    "autoscaling:Describe*",
+    "cloudformation:DescribeStacks",
+    "cloudformation:DescribeStackEvents",
+    "cloudformation:DescribeStackResources",
+    "cloudformation:GetTemplate",
+    "cloudfront:Get*",
+    "cloudfront:List*",
+    "cloudwatch:Describe*",
+    "cloudwatch:Get*",
+    "cloudwatch:List*",
+    "dynamodb:GetItem",
+    "dynamodb:BatchGetItem",
+    "dynamodb:Query",
+    "dynamodb:Scan",
+    "dynamodb:DescribeTable",
+    "dynamodb:ListTables",
+    "ec2:Describe*",
+    "elasticache:Describe*",
+    "elasticbeanstalk:Check*",
+    "elasticbeanstalk:Describe*",
+    "elasticbeanstalk:List*",
+    "elasticbeanstalk:RequestEnvironmentInfo",
+    "elasticbeanstalk:RetrieveEnvironmentInfo",
+    "elasticloadbalancing:Describe*",
+    "iam:List*",
+    "iam:Get*",
+    "route53:Get*",
+    "route53:List*",
+    "rds:Describe*",
+    "s3:Get*",
+    "s3:List*",
+    "sdb:GetAttributes",
+    "sdb:List*",
+    "sdb:Select*",
+    "ses:Get*",
+    "ses:List*",
+    "sns:Get*",
+    "sns:List*",
+    "sqs:GetQueueAttributes",
+    "sqs:ListQueues",
+    "sqs:ReceiveMessage",
+    "storagegateway:List*",
+    "storagegateway:Describe*"
+  ]
+}
+
+resource "aws_iam_group_policy" "my_developer_policy" {
```

Comment:
can we fix module names, if there is no special meaning in resource lets set it this:
aws_iam_group_policy.my_developer_policy => aws_iam_group_policy.this
aws_iam_group.read_only => aws_iam_group.this


why we need  attach_users_to_group options? Cant just have empty "users" variable for this?


please rename police_action => policy_action